models/engine/file_storage.py

```diff
@@ -68,3 +68,17 @@ def delete(self, obj=None):
     def close(self):
         """call reload() method for deserializing the JSON file to objects"""
         self.reload()
+       
+    def get(self, cls, id):
```

Comment:
Thanks, I needed these methods in the file_storage.py to be able to test with it